package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "lint": "mocha tests/lint-test",
-    "test": "npm run lint && mocha tests node-tests --recursive"
+    "test": "mocha tests node-tests --recursive",
+    "test:debugger": "mocha tests node-tests --recursive --inspect-brk"
```

Comment:
you can achieve the same thing with `yarn test --inspect-brk`. I'm not sure we need a dedicated script for this.

Reply:
Thank you for your (very fast) review @Turbo87 
Good to know the tip!

Reply:
I'll close this PR then and open a new one to fix https://github.com/ember-cli/ember-cli-blueprint-test-helpers/issues/185